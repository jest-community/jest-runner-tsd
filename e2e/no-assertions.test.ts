import { expect, test } from '@jest/globals';
import runJest from './runJest';

test('runs a test file with no assertions', async () => {
  expect(await runJest('no-assertions')).toMatchSnapshot();
});
