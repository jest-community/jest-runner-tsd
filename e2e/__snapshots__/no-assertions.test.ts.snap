// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`runs a test file with no assertions 1`] = `
"FAIL ./index.test.ts
  ✕ tsd typecheck
  ● tsd typecheck
    Argument of type 'number' is not assignable to parameter of type 'string'.
    > 4 | concat(1, 2);
        |        ^
      at index.test.ts:4:8
    Expected 2 arguments, but got 1.
    > 5 | concat([1, 2]);
        | ^
      at index.test.ts:5:1
Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:
Ran all test suites.
"
`;
